src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#logo" class="logo"><img src="images/logo.png" alt="logotip"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="link is-active">
```

Comment:
link is not block it's element of nav block. Fix it everywhere

```suggestion
            <a href="#apple" class="nav__link is-active">
```